tuple-table/src/main/java/com/iluwatar/tuppletable/App.java

```diff
@@ -0,0 +1,27 @@
+package com.iluwatar.tuppletable;
+
+
+import java.sql.SQLException;
+
```

Comment:
Above the `App`, add Javadoc to briefly explain the pattern and how the example implements it

Reply:
added.